pkg/clusteragent/admission/mutate/autoinstrumentation/target_mutator.go

```diff
@@ -92,8 +92,10 @@ func NewTargetMutator(config *Config, wmeta workloadmeta.Component) (*TargetMuta
 
 		// Get the library versions to inject. If no versions are specified, we inject all libraries.
 		var libVersions []libInfo
+		useLanguageDetection := false
```

Comment:
rename to `languageDetectionEligible` or something because `use` implies that it _will_ but this means that it _may_, which was confusing to read.